### DISCUSSION TITLE
hcxdumptool and TX power

### DISCUSSION THREAD

**Discussion by ZerBea** [ZerBea/hcxdumptool#419] [General]
A good directional antenna is much better than high TX power, because it amplifies the signal in both directions (client to hcxdumptool and hcxdumptool, to client).
As long as you don't want to (stupid) jam a frequency, TX power of the target and the attack device should be the same and the antenna should be a panel or a grid/dish.
All stations on a channel monitor the channel. If the channel is busy, a station waits until the medium becomes free. Regardless if the TX power is 100W or 100mW the station must wait. An unmodified firmware does not allow to transmit if the channel is busy.

**Comment by IncSTK** [comment]
Can you recommend any specific brand/model of antenna which in your experience performs well? Thanks.

**Reply by ZerBea** [reply]
These conditions should met:

Panel
gain 14 ~ 16dbi
beam width 24 ~ 28 degrees

Grid
gain 22 ~ 26 dbi
beam width 10 ~ 12 degrees

e.g.
https://github.com/ZerBea/hcxdumptool/wiki/Antennas
